modules/persistent-storage-csi-tp-enable.adoc

```diff
@@ -0,0 +1,102 @@
+
+. Enable feature gates (see xref:../../nodes/clusters/nodes-cluster-enabling-features.adoc#[Enabling features using feature gates]).
++
+[IMPORTANT]
+====
+After turning Technology Preview features on using feature gates, they cannot be turned off and cluster upgrades are prevented.
+====
+. Verify the cluster operator storage:
++
+[source,terminal]
+----
+$ oc get co storage
+----
++
+[source, terminal]
+----
+NAME    VERSION                             AVAILABLE   PROGRESSING DEGRADED    SINCE
+storage 4.8.0-0.nightly-2021-04-30-201824   True        False       False       4h26m
+----
++
+* `AVAILABLE` should be "True".
+* `PROGRESSING` should be "False".
+* `DEGRADED` should be "False".
+
+. Verify the status of the pods in the `openshift-cluster-csi-drivers` namespace to verify that they are running:
++
+[source,terminal]
+----
+$ oc get pod -n openshift-cluster-csi-drivers
+----
++
+ifdef::vsphere[]
+[source, terminal]
+----
+NAME                                                    READY   STATUS  RESTARTS    AGE
+vmware-vsphere-csi-driver-controller-5646dbbf54-cnsx7   9/9     Running 0           4h29m
+vmware-vsphere-csi-driver-node-ch22q                    3/3     Running 0           4h37m
+vmware-vsphere-csi-driver-node-gfjrb                    3/3     Running 0           4h37m
+vmware-vsphere-csi-driver-node-ktlmp                    3/3     Running 0           4h37m
+vmware-vsphere-csi-driver-node-lgksl                    3/3     Running 0           4h37m
+vmware-vsphere-csi-driver-node-vb4gv                    3/3     Running 0           4h37m
+vmware-vsphere-csi-driver-operator-7c7fc474c-p544t      1/1     Running 0           4h29m
+----
+endif::vsphere[]
+
+ifdef::azure[]
+[source, terminal]
+----
+NAME                                                    READY   STATUS  RESTARTS    AGE
+microsoft-azure-csi-driver-controller-5646dbbf54-cnsx7  9/9     Running 0           4h29m
+microsoft-azure-csi-driver-node-ch22q                   3/3     Running 0           4h37m
+microsoft-azure-csi-driver-node-gfjrb                   3/3     Running 0           4h37m
+microsoft-azure-csi-driver-node-ktlmp                   3/3     Running 0           4h37m
+microsoft-azure-csi-driver-node-lgksl                   3/3     Running 0           4h37m
+microsoft-azure-csi-driver-node-vb4gv                   3/3     Running 0           4h37m
+microsoft-azure-csi-driver-operator-7c7fc474c-p544t     1/1     Running 0           4h29m
+----
+endif::azure[]
+
+. Verify that the storage class is installed:
++
+[source,terminal]
+----
+$ oc get storageclass
+----
++
+ifdef::vsphere[]
+[source,terminal]
+----
+NAME            PROVISIONER                     RECLAIMPOLICY   VOLUMEBINDINGMODE       ALLOWVOLUMEEXPANSION    AGE
+thin (default)  kubernetes.io/vsphere-volume    Delete          Immediate               false                   5h43m
+thin-csi        csi.vsphere.vmware.com          Delete          WaitForFirstConsumer    true                    4h38m <1>
```

Comment:
minor nit: I think `AllowVolumeExpansion` column will be `false` for CSI driver.